src/ripple/rpc/impl/GRPCHelpers.cpp

```diff
@@ -770,6 +770,14 @@ populateIndexes(T& to, STObject const& from)
     populateProtoVec256([&to]() { return to.add_indexes(); }, from, sfIndexes);
 }
 
+template <class T>
+void
+populateTokenOffers(T& to, STObject const& from)
```

Comment:
Rename to `populateNFTOffers`. OK, I'll stop adding comment for functions to rename. There's a general comment of name nft things "nft" and don't use "token" or make the function too generic sounding.

Reply:
This rename I'm not so confident about.  The problem is that `populateTokenOffers()` is supposed to echo the name of the `SField`, which is `TokenOffers`.  I'd prefer not to rename this function unless we also rename the `SField`.  Consistent naming is interconnected...